plugins/version-control-systems/git/src/main/kotlin/Git.kt

```diff
@@ -104,6 +104,9 @@ class Git(
 ) : VersionControlSystem() {
     companion object {
         init {
+            // Work around a bug in Apache MINA sshd, see https://github.com/eclipse-jgit/jgit/issues/135.
+            System.setProperty("java.security.egd", "file:/dev/./urandom")
```

Comment:
Should this only be executed if the OS is Linux?

Reply:
Good point, done, and added an explanation to the commit message.